fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/values/translation/TrivialTranslator.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * SimpleTranslator.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2015-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.record.query.plan.cascades.values.translation;
+
+import com.apple.foundationdb.record.query.plan.cascades.AliasMap;
+import com.apple.foundationdb.record.query.plan.cascades.CorrelationIdentifier;
+import com.apple.foundationdb.record.query.plan.cascades.TranslationMap;
+import com.apple.foundationdb.record.query.plan.cascades.values.Value;
+
+import javax.annotation.Nonnull;
+
+/**
+ * A {@code Translator} that simply translates a given {@code Value} by replacing its {@code queryCorrelation} with the
+ * specified {@code candidateCorrelation}. It does not perform any sub-{@link Value} replacement.
+ */
+public class TrivialTranslator extends Translator {
+
+    @Nonnull
+    private final AliasMap translationAliasMap;
```

Comment:
Why do we keep both `translationAliasMap` and `translationMap` around? I don't think I like the redundancy of this.

Reply:
Translators are removed.